libethash-cl/CLMiner.cpp

```diff
@@ -731,14 +730,19 @@ bool CLMiner::init(const h256& seed)
 		m_dagKernel.setArg(2, m_dag);
 		m_dagKernel.setArg(3, ~0u);
 
+		auto startDAG = std::chrono::steady_clock::now();
 		for (uint32_t i = 0; i < fullRuns; i++)
 		{
 			m_dagKernel.setArg(0, i * m_globalWorkSize);
 			m_queue.enqueueNDRangeKernel(m_dagKernel, cl::NullRange, m_globalWorkSize, m_workgroupSize);
 			m_queue.finish();
-			cllog << "DAG" << int(100.0f * i / fullRuns) << '%';
 		}
+		auto endDAG = std::chrono::steady_clock::now();
 
+		auto dagTime = std::chrono::duration_cast<std::chrono::milliseconds>(endDAG-startDAG);
+		char gb[16];
+		sprintf(gb, "%.2f", (float)dagSize / (1024 * 1024 * 1024));
```

Comment:
WTF? Just do `cnote << std::setprecision(2) << f << " GB of DAG..."`

Reply:
Not sure why this is showing under this pull as I did it as a separate pull request.
As for why, some of us learned C++ before std::setprecision() existed; I did most of my C++ development 20-30 years ago, having gotten started with cfront.
Anyway, I'll update the code to use setprecision shortly.

Reply:
And it seems cnote doesn't work with the precision iomanip:
"2.27344  GB of DAG data generated in 3886 ms"
I'm not about to fix cnote, so I'll just leave it as the default float output.